ErsatzTV.Scanner/Application/Plex/Commands/SynchronizePlexLibraryByIdHandler.cs

```diff
@@ -66,6 +66,13 @@ public class SynchronizePlexLibraryByIdHandler : IRequestHandler<SynchronizePlex
                         parameters.Library,
                         parameters.DeepScan,
                         cancellationToken),
+                LibraryMediaKind.OtherVideos =>
```

Comment:
This is a good idea, but without the changes recommended above, this path isn't hit; library media kind is still movies. Also, a different scanner may be needed to support other video fallback metadata so it's consistent with local other video libraries.